### PR TITLE
This PR has a code to automate user stories - 531, 569, 456

### PR DESCRIPTION
**Pull request by chiragkhimani**
Everything has been verified and we also have green report

### CODE REVIEW

File: src/main/java/com/automation/ForgetPasswordFeature.java [hunk 1]
```diff
@@ -1,5 +1,8 @@
 package com.automation;
 
 public class ForgetPasswordFeature {
-
+	public static void main(String[] args) {
+		System.out.println("yess! I started another feature");
```
Comment (chiragkhimani) [inline src/main/java/com/automation/ForgetPasswordFeature.java:5]: why you need this line?

Comment (sigabyte) [reply src/main/java/com/automation/ForgetPasswordFeature.java:5]: Manager insists for it!

Comment (chiragkhimani) [reply src/main/java/com/automation/ForgetPasswordFeature.java:5]: This is not right approach btw, you should reconsider assigning variable first

File: src/main/java/com/automation/HomeFeatureCode.java [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 public class HomeFeatureCode {
 	public static void main(String[] args) {
-
+		System.out.println("This is my homepage code");
+		System.out.println("Let me complete fast before manager come");
```
Comment (sigabyte) [inline src/main/java/com/automation/HomeFeatureCode.java:6]: Sorry! you are late to complete before manager arrival. This line need to be updated!

File: src/main/java/com/automation/LoginTest.java [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 public class LoginTest {
 	public static void main(String[] args) {
-		
+		System.out.println("1st line of code in login");
+		System.out.println("I am enjoying this code of login");
+		System.out.println("Remote! This is for you");
```
Comment (sagambay) [inline src/main/java/com/automation/LoginTest.java:7]: good job!

File: src/main/java/com/automation/ForgetPasswordFeature.java (hunk 1 above)
Comment (SaidShuhrat) [reply src/main/java/com/automation/ForgetPasswordFeature.java:5]: I am also here to finish project till Manager comes

File: src/main/java/com/automation/HomeFeatureCode.java (hunk 2 above)
Comment (sagambay) [inline src/main/java/com/automation/HomeFeatureCode.java:6]: good luck!

File: src/main/java/com/automation/ForgetPasswordFeature.java (hunk 1 above)
Comment (chiragkhimani) [reply src/main/java/com/automation/ForgetPasswordFeature.java:5]: btw who is your manager?

Comment (MayramShadieva) [reply src/main/java/com/automation/ForgetPasswordFeature.java:5]: Manager will be late, take your time

Comment (MayramShadieva) [reply src/main/java/com/automation/ForgetPasswordFeature.java:5]: she is doing her nails

**Review by MayramShadieva — APPROVED** [review]
Great job

**Review by sagambay — APPROVED** [review]
need some practice

**Review by SaidShuhrat — APPROVED** [review]

**Comment by chiragkhimani** [comment]
image

**Pull request merged**